bitrise.yml

```diff
@@ -1458,7 +1458,7 @@ trigger_map:
 - push_branch: release/v116
   pipeline: pipeline_multiple_shards
 - pull_request_target_branch: main
-  pipeline: pipeline_multiple_shards
+  pipeline: perfherder-test
```

Comment:
@isabelrios Is this the way to select `perfheder-test` to see if the performance tests will work?

Reply:
Use workflow instead of pipeline